src/server/api.js

```diff
@@ -44,11 +50,11 @@ export const CreateGame = async (db, game, numPlayers, setupData) => {
   });
 
   for (let playerIndex = 0; playerIndex < numPlayers; playerIndex++) {
-    const credentials = createCredentials();
+    const credentials = lobbyConfig.shortid();
```

Comment:
Shall we call this option something more generic like `uuid`?  If someone overrides it to generate a longer ID, then `shortid` won't be a particularly relevant name.